package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/UrpShaderImporter.cs

```diff
@@ -0,0 +1,178 @@
+
+
+using UnityEngine;
+
+namespace Unity.Formats.USD
+{
+    public class UrpShaderImporter : ShaderImporterBase
+    {
+                private static Material ChannelCombinerMat;
+
+        public UrpShaderImporter(Material material) : base(material)
+        {
+        }
+
+        public override void ImportFromUsd()
+        {
+            Material mat = Material;
+
+            if (DiffuseMap)
+            {
+                Debug.Log("here");
+                mat.SetTexture("_BaseMap", DiffuseMap);
+                mat.SetColor("_BaseColor", Color.white);
+            }
+            else
+            {
+                mat.SetColor("_BaseColor", Diffuse.GetValueOrDefault(mat.color));
+            }
+
+            // TODO: What about opacity map?
```

Comment:
It might need to be merged into the base map as the alpha channel - the other importers have examples of merging texture channels on import